lib/animation.dart

```diff
@@ -15,6 +15,8 @@ class Frame {
   Frame(this.sprite, this.stepTime);
 }
 
+typedef OnCompleteAnimation = Function();
```

Comment:
Change the `Function()` to `void Function()` so we can have the type check on the return type.

Reply:
ok will do it.